src/themes/OLH/templates/journal/article.html

```diff
@@ -89,7 +89,7 @@ <h3>{{ article.title|safe }}</h3>
         </section>
         {% for author in article.frozen_authors.all %}
             {% if author.biography %}
-                <div class="dropdown-pane author-info" data-dropdown id="author{{ forloop.counter }}" data-hover="true"
+                <div class="dropdown-pane author-info proofing-test-include" data-dropdown id="author{{ forloop.counter }}" data-hover="true"
```

Comment:
I don't see this CSS class defined anywhere, where is it coming from?

Reply:
I just saw https://github.com/BirkbeckCTP/typesetting/pull/185/files and understood what you are trying to do here.

It'd be best to give these nodes an `id` attribute and then have the test define what elements it should see and which it should, rather than bundling this ghost articles into the live article template

(e.g.: give the how to cite block an `id="how-to-cite"`)

Reply:
Sure--I've made the change